index.js

```diff
@@ -230,12 +237,15 @@ Use getArtistByIndex to do the following:
 
 🌟 EXAMPLE: if getArtistByIndex is invoked with the artists array and the number 0, it will return `the artist at index 0 is Amedeo Modigliani` */
 
-function getArtistByIndex(/*Your Code Here*/) {
-  /*Your Code Here*/
+function getArtistByIndex(array, index) {
+  for (let i = 0; i < array.length; i++) {
```

Comment:
Why are you using a for loop here? Since you're given the index do you need to loop through everything in the array?